components/form-control/script.js

```diff
@@ -238,11 +352,108 @@ class GamefaceFormControl extends HTMLElement {
         }
     }
 
+    /**
+     * Checks the validator method
+     * @param {{method:()=>boolean}} validator
```

Comment:
Use **Validator** defined as a `@typedef` from the previous comment about the JSDoc
or
a valid JSDoc comment.

Reply:
FIxed